earth_enterprise/rpms/build.gradle

```diff
@@ -62,15 +81,17 @@ if (new File('build.properties').exists()) {
     }
 }
 
-def openGeeVersionFile = new File("../src/NATIVE-DBG-x86_64/gee_long_version.txt")
+def openGeeVersionFile = new File( gee_long_version_txt )
```

Comment:
Minor: I don't see the need to generate a `File` containing the path to `gee_long_version.txt`, and a `string` containing the same thing.
- [ ] Consider using one variable with the path to `gee_long_version.txt`.